cmds/core/nohup/nohup.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// nohup – invoke a utility immune to hangups.
+//
+// Synopsis:
+//
+//	nohup <command> [args...]
+//
+
```

Comment:
no empty line here -- it's only a package comment if it directly precedes the `package main` statement